project_event/i18n/fr.po

```diff
@@ -670,3 +717,6 @@ msgstr "Wizard de création d'événement à partir d'un modèle"
 #: model:ir.model,name:project_event.model_project_task_wizard
 msgid "Wizard Event Task Creation from template"
 msgstr "Wizard de création de tâche d'événement à partir d'un modèle"
+
```

Comment:
@: This translation  is from the header comment text  """Wizard Event Creation from Template""" in the wizard file "project_event_wizard.py" . Same text available in the other files "project_event_wizard.py" and "project_task_wizard.py" . Please update me all the required translation for the project_event module so that I can update accordingly in this pull request